spec/chartmogul/utils/json_parser_spec.rb

```diff
@@ -23,5 +23,11 @@
       result = described_class.typecast_custom_attributes(attr: '2016-02-01T0000:00.000Z')
       expect(result[:attr]).to be_instance_of(String)
     end
+
+    it 'allows skipping camel case conversion' do
+      json = {'aKey': 'a_value', 'bKey': 'b_value'}.to_json
```

Comment:
What happens if we will send `{ 'aKey': 'a_value', 'a_key': 'another_value'}` ?

Reply:
If skip `skip_case_conversion` is true, it will become `{aKey: 'a_value', a_key: 'another_value'}`.
If it is set to false, the keys will be merged into one. Since it is false by default, this will function as before.